src/midi_Defs.h

```diff
@@ -164,8 +164,22 @@ enum MidiControlChangeNumber: uint8_t
     GeneralPurposeController2   = 17,
     GeneralPurposeController3   = 18,
     GeneralPurposeController4   = 19,
-
+    // CC20 to CC31 undefined
+    BankSelectLSB               = 32,
+    ModulationWheelLSB          = 33,
+    BreathControllerLSB         = 34,
+    // CC35 undefined
+    FootControllerLSB           = 36,
+    PortamentoTimeLSB           = 37,
     DataEntryLSB                = 38,
+    ChannelVolumeLSB            = 39,
+    BalanceLSB                  = 40,
+    // CC41 undefined
+    PortamentoTimeLSB           = 42,
```

Comment:
Duplicate `PortamentoTimeLSB` should be `PanLSB`

Reply:
Whoops, fixed